### PR TITLE
Partial NetBSD support

### PR DESCRIPTION
**Pull request by battleblow**
These changes add partial support for NetBSD 8.0/amd64.  There are other changes necessary, but these are ready to be reviewed and merged.

### CODE REVIEW

**Comment by battleblow** [comment]
I've compile tested these on FreeBSD, but not OpenBSD

**Review by bsdkurt — APPROVED** [review]
Reads well to me and test build on OpenBSD was successful.

File: src/java.base/bsd/native/libjava/ProcessHandleImpl_bsd.c [hunk 1]
```diff
@@ -23,6 +23,10 @@
  * questions.
  */
 
+#ifdef __NetBSD__
+#define _KMEMUSER
```
Comment (krytarowski) [inline src/java.base/bsd/native/libjava/ProcessHandleImpl_bsd.c]: Why this? Do you run the process as root and read kernel memory?

Comment (bsdkurt) [reply src/java.base/bsd/native/libjava/ProcessHandleImpl_bsd.c]: It looks like NetBSD transitioned to KERN_PROC2 with struct kinfo_proc2 which is similar to the 6 entry mib that OpenBSD uses as well.

Comment (battleblow) [reply src/java.base/bsd/native/libjava/ProcessHandleImpl_bsd.c]: This is a fair question.  Without this there are compilation errors, but let me see if I can find a better solution.

Comment (battleblow) [reply src/java.base/bsd/native/libjava/ProcessHandleImpl_bsd.c]: I've checked again and the final version of these changes didn't need _KMEMUSER.  I just needed to include <sys/resource.h>.  I've done so and removed the _KMEMUSER define.

**Comment by battleblow** [comment]
> Reads well to me and test build on OpenBSD was successful.

Thanks Kurt!  I'm going to go ahead and merge now.

**Pull request merged**